app/forms/phone_helpers.rb

```diff
@@ -0,0 +1,22 @@
+module PhoneHelpers
+  def require_phone_confirmation?
+    phone_changed?
+  end
+
+  def phone_changed?
+    @phone_changed == true
+  end
+
+  private
+
+  def check_phone_change(params)
+    formatted_phone = params[:phone].phony_formatted(
+      format: :international, normalize: :US, spaces: ' '
+    )
+
+    return unless formatted_phone != @user.phone
+
+    @phone_changed = true
+    self.phone = formatted_phone
+  end
+end
```

Comment:
Why write a separate module that's only included in one form? Also, this refactoring doesn't seem to be related to this PR. It looks like you did this to make some changes in `edit_phone_controller.rb`, which don't look like they're necessary. You're calling `@update_user_phone_form.phone_changed?`, and then `@update_user_phone_form.require_phone_confirmation?`, which both do the same thing. In this module, `require_phone_confirmation?` acts as an alias for `phone_changed?`.

So, it looks like these changes are not needed.